independent-publisher-connectors/Zoom Meetings/apiDefinition.swagger.json

```diff
@@ -0,0 +1,141 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zoom Meetings",
+    "description": "This Connector will help to automate Zoom Meeting Operations",
+    "version": "1.0"
+  },
+  "host": "api.zoom.us",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v2/users/me/meetings": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Meetings",
+        "description": "Gives the Upcoming Meetings",
+        "operationId": "GetMetings",
+        "parameters": []
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create Meeting",
+        "description": "For Creating Meetings",
+        "operationId": "CreateMeeting",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "topic": {
+                  "type": "string",
+                  "description": "topic"
+                },
+                "type": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "type"
+                },
+                "start_time": {
+                  "type": "string",
+                  "description": "start_time"
+                },
+                "duration": {
+                  "type": "string",
+                  "description": "duration"
+                },
+                "settings": {
+                  "type": "object",
+                  "properties": {
+                    "host_video": {
+                      "type": "boolean",
+                      "description": "host_video"
+                    },
+                    "participant_video": {
+                      "type": "boolean",
+                      "description": "participant_video"

```

Comment:
@A-Deekshith All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Reply:
@A-Deekshith Please address and fix this comment.